examples/minimal/package.json

```diff
@@ -3,19 +3,17 @@
   "private": true,
   "scripts": {
     "build": "pnpm recursive run build",
-    "dev": "run-pty % pnpm dev:node % pnpm dev:client-react % pnpm dev:contracts",
+    "dev": "run-pty % pnpm dev:client-react % pnpm dev:contracts",
```

Comment:
`run-pty` is still here but removed from deps below?

Reply:
good catch! fixed